### PR TITLE
Update planning publisher

### PR DESCRIPTION
**Pull request by ride90**

### CODE REVIEW

File: client/extensions/helloWorld2/npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```
Comment (petrjasek) [inline client/extensions/helloWorld2/npm-debug.log]: this file shouldn't be there we can probably ignore `*.log`

Comment (tomaskikutis) [reply client/extensions/helloWorld2/npm-debug.log]: `npm-debug.log` shouldn't be committed either

File: server/Procfile [hunk 2]
```diff
@@ -3,3 +3,4 @@ wamp: python3 -u ws.py
 work: celery -A worker worker
 beat: celery -A worker beat --pid=
 capi: gunicorn -c gunicorn_config.py content_api.wsgi
+highcharts: python3 -u -m analytics.reports.highcharts_server
```
Comment (petrjasek) [inline server/Procfile]: seems like analytics is not installed so this won't work

File: client/extensions/helloWorld2/npm-debug.log (hunk 1 above)
Comment (ride90) [reply client/extensions/helloWorld2/npm-debug.log]: my bad, I did not notice it somehow..

File: server/requirements.txt [hunk 3]
```diff
@@ -2,6 +2,5 @@ gunicorn==19.7.1
 honcho==1.0.1
 newrelic>=2.66,<2.67
 
--e git+git://github.com/superdesk/superdesk-core.git@f38aaee#egg=Superdesk-Core
--e git+git://github.com/superdesk/superdesk-planning.git@master#egg=superdesk-planning
-
+-e git+git://github.com/superdesk/superdesk-core.git@v1.32.5#egg=Superdesk-Core
```
Comment (petrjasek) [inline server/requirements.txt]: you can use `superdesk-core>=1.32.5,<1.33` for this one. but maybe it should just use latest stable

**Pull request merged**